packages/template-retail-react-app/app/commerce-api/hooks/useBasket.js

```diff
@@ -365,6 +365,11 @@ export default function useBasket(opts = {}) {
              */
             async createOrder() {
                 const response = await api.shopperOrders.createOrder({
+                    // We send the SLAS usid via this header. This is required by ECOM to map
+                    // Einstein events sent via the API with the finishOrder event fired by ECOM
+                    // when an Order transitions from Created to New status.
+                    // Without this, various order conversion metrics will not appear on reports and dashboards
+                    headers: {_sfdc_customer_id: api.auth.usid},
```

Comment:
Do we want to add a test to confirm the expected header is sent when creating orders?

Reply:
I've pushed a change to the existing test to add the header. However, I'm not sure of a good way to test this change via unit test. The header does not modify the order response in any way.

I could make setting the header a requirement in OcapiShopperOrders but I don't know of this is a good change (ie. sites that don't use Einstein will have no need to set this header).